[rank: 0] Global seed set to 42
Traceback (most recent call last):
  File "/home/u563586/tft_project/src/main_tft_short.py", line 17, in <module>
    with open("config.yaml") as file:
FileNotFoundError: [Errno 2] No such file or directory: 'config.yaml'
